.gitignore

```diff
@@ -0,0 +1,3 @@
+learning/
+
```

Comment:
Please add db.sqlite3, **/__pycache__/** to .gitignore.
You can remove the added files using git rm
e.g. git rm db.sqlite3
git rm storefront/storefront/playground/__pycache__/